.circleci/config.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - checkout
       - run:
           name: Update npm
-          command: 'sudo npm install -g npm@v10.1.0'
+          command: 'sudo npm install -g npm@latest'
```

Comment:
Amended to mirrored to install latest as configured in several other applications to ensure this is picked up alongside any renovate dependancy updates